p2p/net/swarm/addr/addr_test.go

```diff
@@ -39,9 +39,13 @@ func TestFilterAddrs(t *testing.T) {
 		if AddrUsable(a, false) {
 			t.Errorf("addr %s should be unusable", a)
 		}
+		/* This doesnt make sense anymore
+		With utp allowed, udp gets 'allowed' as a partial address.
```

Comment:
@jbenet i'm not sure how this needs to work. If we allow `/ip4/0.0.0.0/udp/1234/utp` then `/ip4/0.0.0.0/udp/1234` is also allowed as a 'partial' address